captcha/image.py

```diff
@@ -31,25 +31,27 @@
 
 
 class _Captcha(object):
-    def generate(self, chars):
+    def generate(self, chars, format='jpeg'):
```

Comment:
If you must add the format parameter, you should use `png`(which is the formal format) as the default value.